components/Buttons/src/MDCButton.m

```diff
@@ -101,6 +101,7 @@ @interface MDCButton () {
 
 @implementation MDCButton
 
+@synthesize mdc_overrideBaseElevation = _mdc_overrideBaseElevation;
```

Comment:
should we initialize this to -1?

Reply:
Yeah - good catch.

Reply:
Done. I'll push up a PR for other components